.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,36 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue related to manifests
```

Comment:
Discussion: do we really need this?

I think when people come and want to create an issue, the corresponding group that should handle it should be:
* application owners if it's application specific (including manifest issues)
* distribution owners if it's distribution specific

I don't think there's a case they need to go to manifests WG.

Reply:
Agree